mutmut/__init__.py

```diff
@@ -35,6 +35,8 @@
 __version__ = '2.3.0'
 
 
+if os.getcwd() not in sys.path:
+    sys.path.insert(0, os.getcwd())
```

Comment:
``__init__`` is executed first even if ``__main__`` is the script that is actually passed to Python (or the entry point in this case), so we have to move this from ``__main__`` to here.